data_model/last_temp.py

```diff
@@ -1,9 +1,16 @@
 import requests
 from data_model import model
+from data_model.tempcron import schedule_cron_job
 
-r = requests.get("http://localhost:5010/read")
-x = r.json().get('data')
 
-last_temp = x.get('temperature')
-thermostat = model.Device(1, 2, 3, "Benny's Thermostat", "thermostat", 1, last_temp, 20, 0)
+def get_last_temp():
+    r = requests.get("http://localhost:5010/read")
+    x = r.json().get('data')
+    thermostat = model.Device(1, 2, 3, "Benny's Thermostat", "thermostat", 1, 25.0, 20, 0)
+    print("last temperature = %.2f" % thermostat.last_temp)
```

Comment:
At the moment the cron just prints temperature readings to demonstrate it works.